CONTRIBUTING.md

```diff
@@ -25,7 +25,7 @@ Thanks for your contributions!
 
 ### Code style
 
-The coding style suggested by the rust community and clippy project. See the [style doc](https://aturon.github.io/README.html) and [clippy](https://github.com/Manishearth/rust-clippy) for details.
+The coding style suggested by the rust community and clippy project. See the [style doc](https://doc.rust-lang.org/1.0.0/style/README.html) and [clippy](https://github.com/Manishearth/rust-clippy) for details.
```

Comment:
Now official guide becomes https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/guide.md.

Reply:
> Now official guide becomes https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/guide.md.

Thank you. I‘ve been learn rust about a week, this guide very useful to me.

Reply:
should we update to the new official guide here? @BusyJay

Reply:
Yes, please update it with the provided link.

Reply:
Friendly ping @NOVA-ME

Reply:
> Friendly ping @NOVA-ME

Everything is ok. I'm sry reply so late. A little busy these days.